packages/teleport/src/Kube/Kube.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { Box, Indicator } from 'design';
+import { Danger } from 'design/Alert';
+import { Attempt } from 'shared/hooks/useAttemptNext';
+import KubeList from 'teleport/components/KubeList';
+import { Kube as KubeProps } from 'teleport/services/kube';
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout';
+
+type Props = {
+  kubes: KubeProps[];
+  attempt: Attempt;
+};
+
+export default function Kube(props: Props) {
```

Comment:
Lets rename this to `Kubes` (including the filename), so we can use `Kube` for type instead of `KubeProps`